addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,12 +1,49 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import api, fields, models, _
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
 
+    slide_ids = fields.One2many(
+        'slide.slide', 'survey_id', string="Certification Slides",
+        help="The slides this survey is linked to through the e-learning application")
+    slide_channel_ids = fields.One2many(
+        'slide.channel', string="Certification Courses", compute='_compute_slide_channel_ids',
+        help="The courses this survey is linked to through the e-learning application",
+        groups='website_slides.group_website_slides_officer')
+    slide_channel_count = fields.Integer("Courses Count", compute='_compute_slide_channel_ids', groups='website_slides.group_website_slides_officer')
+
+    @api.depends('slide_ids.channel_id')
+    def _compute_slide_channel_ids(self):
```

Comment:
Just for clarity I would rename it _compute_slide_data or _compute_slide_statistics, because it computes values for two fields.

Reply:
Changed to _compute_slide_channel_data :)